srv/pom.xml

```diff
@@ -61,6 +67,16 @@
 			<artifactId>scp-cf</artifactId>
 		</dependency>
 
+		<!-- CLOUD SDK for on-premise connectivity -->
```

Comment:
@beckermarc: I wonder if we still need it. I could retry without.

Reply:
I guess we do, if you want to connect to on premise systems. But you can try out what happens without these dependencies. AFAIK they are however responsible for implementing the connectivity service stuff, which is not required for cloud S/4 hana systems. Not sure if the setup to connect to an on-premise S/4 needs to be part of the sample though.